edgelet/build/windows/build.ps1

```diff
@@ -1,23 +1,80 @@
 # Copyright (c) Microsoft. All rights reserved.
 
 param(
-    [switch]$Release
+    [switch]$Release,
+    [switch]$Arm
 )
 
 # Bring in util functions
 $util = Join-Path -Path $PSScriptRoot -ChildPath "util.ps1"
 . $util
 
 # Ensure rust is installed
-Assert-Rust
+Assert-Rust -Arm:$Arm
+
+$cargo = Get-CargoCommand -Arm:$Arm
+
+$ErrorActionPreference = 'Continue'
+
+# arm build has to use a few private forks of dependencies instead of the public ones, in order to to this, we have to 
+# 1. append a [patch] section in cargo.toml to use crate forks
+# 2. run cargo update commands to force update cargo.lock to use the forked crates
+# after these two steps, running cargo build for arm should not show any warning such as "Patch 'crate-x' was not used in the crate graph"
+if($Arm)
+{
+    $edgefolder = Get-EdgeletFolder
+    Set-Location -Path $edgefolder
+
+    $ForkedCrates = @"
+
+[patch.crates-io]
+backtrace = { git = "https://github.com/philipktlin/backtrace-rs", branch = "arm" }
+cmake = { git = "https://github.com/philipktlin/cmake-rs", branch = "arm" }
+dtoa = { git = "https://github.com/philipktlin/dtoa", branch = "arm" }
+iovec = { git = "https://github.com/philipktlin/iovec", branch = "arm" }
+mio = { git = "https://github.com/philipktlin/mio", branch = "arm" }
+miow = { git = "https://github.com/philipktlin/miow", branch = "arm" }
+serde-hjson = { git = "https://github.com/philipktlin/hjson-rust", branch = "arm" }
+winapi = { git = "https://github.com/philipktlin/winapi-rs", branch = "arm/v0.3.5" }
+
+[patch."https://github.com/Azure/mio-uds-windows"]
+mio-uds-windows = { git = "https://github.com/philipktlin/mio-uds-windows", branch = "arm" }
+
+[patch."https://github.com/Azure/hyperlocal-windows"]
```

Comment:
You shouldn't need to patch [`hyperlocal-windows`](https://github.com/philipktlin/hyperlocal-windows/compare/master..arm) and [`tokio-uds-windows`](https://github.com/philipktlin/tokio-uds-windows/compare/master..arm) since they don't have any changes except to remove the `.git` suffix from the URL, which is not significant.

Reply:
see response to the other comment above

Reply:
Same here. I wrote this comment after I tested it on my machine.

Reply:
OK, now I can repro what you suggested locally, pushed a fix and started a new build